src/commands/cmd_tdigest.cc

```diff
@@ -168,7 +168,69 @@ class CommandTDigestAdd : public Commander {
   std::vector<double> values_;
 };
 
+class CommandTDigestMin : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    key_name_ = args[1];
+    return Status::OK();
+  }
+
+  Status Execute(engine::Context &ctx, Server *srv, Connection *conn, std::string *output) override {
+    TDigest tdigest(srv->storage, conn->GetNamespace());
+    TDigestMetadata metadata;
+    auto s = tdigest.GetMetaData(ctx, key_name_, &metadata);
+    if (!s.ok()) {
+      if (s.IsNotFound()) {
+        return {Status::RedisExecErr, errKeyNotFound};
+      }
+      return {Status::RedisExecErr, s.ToString()};
+    }
+
+    if (metadata.total_observations == 0) {
+      return {Status::RedisExecErr, errValueIsNotExists};
```

Comment:
https://github.com/RedisBloom/RedisBloom/blob/f68a72e463b84c2298990295c6a3db4719dc5c7a/src/rm_tdigest.c#L386

What about?

Reply:
Hi @mapleFU ,

Thanks very much for this link.❤️

I had seeking for a long time for redis's native tdigest implementation during my development but didn't find it. 😆

Best Regards,
Edward